python/tchannel/exceptions.py

```diff
@@ -54,6 +54,11 @@ class InvalidMessageException(ProtocolException):
     pass
 
 
+class InvalidEndpointException(ProtocolException):
+    """Represent an message containing invalid endpoint."""
+    pass
```

Comment:
InvalidExceptionEndpoint shouldn't be a ProtocolException.

ProtocolException represents an error caused by protocol error frames.